subprojects/crc32c/meson.build

```diff
@@ -1,12 +1,21 @@
 project(
     'crc32c',
-    ['c']
+    'c'
 )
 
 crc32c = library(
     meson.project_name(),
-    ['crc32c.c'],
-    gnu_symbol_visibility: 'hidden'
+    'crc32c.c',
+    gnu_symbol_visibility: 'hidden',
+    override_options: [
+        # Inline assembly functions cannot be properly LTOed. One workaround
+        # would be to always call crc32c_sw(), but then you would lose out in
+        # the case when LTO is disabled. Another solution would be to mark
+        # assembly function as noinline, but since these files come from
+        # elsewhere, it is probably best to not edit them. For these reasons,
+        # always forcibly disable LTO.
+        'b_lto=false',
+    ]
```

Comment:
So I could move the memlcp functions into say, arch_asm.c or something and then we could use this and not have to rely on noinline being defined...

Reply:
You would have to create a static library with them to get that ^ unless you wanted all of hse to be forced non-LTO. I would argue moving the functions into a memlcp.c is probably the right move though.